### PR TITLE
OLS-705 "Reduce Konflux pipeline execution time for RAG repository", Nvidia GPU support

### PR DESCRIPTION
**Pull request by syedriko**
Nvidia GPU support

JIRA: https://issues.redhat.com/browse/OLS-705 "Reduce Konflux pipeline execution time for RAG repository"

### CODE REVIEW

**Comment by syedriko** [comment]
@asamal4 @tisnik PTAL

**Comment by syedriko** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-rag-content"

File: .tekton/lightspeed-rag-content-pull-request.yaml [hunk 1]
```diff
@@ -238,16 +240,20 @@ spec:
...
-      timeout: "8h0m0s"
+      timeout: "1h0m0s"
       taskRef:
         params:
         - name: name
           value: buildah-remote
-        - name: bundle
-          value: quay.io/konflux-ci/tekton-catalog/task-buildah-remote:0.2@sha256:eb3440a0ea2a1366160d86afdd17e04006aeba58e5b9ef95fbbcb9783cc59a68
+        - name: url
+          value: https://github.com/syedriko/build-definitions
```
Comment (eparis) [inline .tekton/lightspeed-rag-content-pull-request.yaml:249]: what's in github.com/syedriko? is that where it should live?

Comment (syedriko) [reply .tekton/lightspeed-rag-content-pull-request.yaml:249]: For now, until https://github.com/konflux-ci/build-definitions/pull/1529/files merges. Which is when I'll update this to use the updated task-buildah-remote.

**Pull request merged**